nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -134,7 +134,8 @@ nrf_modem_os_trace_put
 
 This function puts the trace string to the desired medium, typically UART.
 However, the medium used to forward and store the traces is up to the implementation and must be initialized correctly before using.
-If you are not interested in traces, they can be ignored, and this function can be empty and simply return.
+Once the traces are processed/stored, the :c:func:`nrf_modem_trace_processed_callback` must called.
```

Comment:
Missing word: must _be_ called.

Perhaps add a sentence about why this is important.

Reply:
Done